src/snyk/common/platform.ts

```diff
@@ -5,6 +5,10 @@ export class Platform {
     return os.platform();
   }
 
+  static getArch(): string {
+    return os.arch();
+  }
+
   static getVersion(): string {
     return `${os.release()}-${os.arch}`;
```

Comment:
does the arch need to be mapped? e.g. from aarch64 to arm64?

Reply:
nope it returns arm64 luckily